pkgs/applications/networking/protonmail-bridge/default.nix

```
@@ -1,73 +1,42 @@
-{ stdenv, fetchurl, lib, qtbase, qtmultimedia, qtsvg, qtdeclarative, qttools, qtgraphicaleffects, qtquickcontrols2, full
-, libsecret, libGL, libpulseaudio, glib, wrapQtAppsHook, mkDerivation }:
+{ lib, buildGoModule, fetchFromGitHub, pkg-config, libsecret }:
 
-let
-  version = "1.2.3-1";
-
-  description = ''
-    An application that runs on your computer in the background and seamlessly encrypts
-    and decrypts your mail as it enters and leaves your computer.
-
-    To work, gnome-keyring service must be enabled.
-  '';
-in mkDerivation {
+buildGoModule rec {
   pname = "protonmail-bridge";
-  inherit version;
+  version = "1.4.5";
 
-  src = fetchurl {
-    url = "https://protonmail.com/download/protonmail-bridge_${version}_amd64.deb";
-    sha256 = "032ggk9fvd19fbsqkzwzwh0hpyg8gpkrin71di7zsx6ias5innw1";
+  src = fetchFromGitHub {
+    owner = "ProtonMail";
+    repo = "proton-bridge";
+    rev = "br-${version}";
+    sha256 = "J7m1JcLrjUw0PIMoRWDXH1hax97/C7zq6Rj2o/SAaYw=";
   };
 
-  sourceRoot = ".";
+  vendorSha256 = "goaMQ4wmUoc/TLsIxmwH9C2VtKsnuwRmAs9C1sGosk0=";
 
-  unpackCmd = ''
-    ar p "$src" data.tar.xz | tar xJ
-  '';
+  nativeBuildInputs = [ pkg-config ];
 
-  installPhase = ''
-    mkdir -p $out/{bin,lib,share}
+  buildInputs = [ libsecret ];
 
-    cp -r usr/lib/protonmail/bridge/protonmail-bridge $out/lib
-    cp -r usr/share $out
-
-    ln -s $out/lib/protonmail-bridge $out/bin/protonmail-bridge
+  buildPhase = ''
+    make build-nogui
```

```suggestion
    make BRIDGE_APP_VERSION='${version}' build-nogui
```

Changes the version from

```
ProtonMail Bridge version 1.4.0-git () 2020-11-09T03:35:16+0000
```

To

```
ProtonMail Bridge version 1.4.5 () 2020-11-09T03:45:57+0000
```